applications/dashboard/views/activity/helper_functions.php

```diff
@@ -151,7 +151,7 @@ function writeActivityComment($comment, $activity) {
             <?php } ?>
             <div class="ItemContent ActivityComment">
                 <?php echo userAnchor($author, 'Title Name'); ?>
-                <div class="Excerpt"><?php echo Gdn_Format::to($comment['Body'], $comment['Format']); ?></div>
+                <div class="Excerpt userContent"><?php echo Gdn_Format::to($comment['Body'], $comment['Format']); ?></div>
```

Comment:
Can the excerpt be something other than plain text? It's not necessary to add `userContent` if it's plain text.